<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MngFormFactory</class>
 <widget class="QDialog" name="MngFormFactory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Manutenção em Fabricantes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/images/PNG/128x128/advancedsettings.png</normaloff>:/images/PNG/128x128/advancedsettings.png</iconset>
  </property>
  <property name="toolTip">
   <string>Permite Adicionar, Alterar ou Excluir um Fabricante...</string>
  </property>
  <widget class="QGroupBox" name="GBoxDataFactory">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>731</width>
     <height>161</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="title">
    <string>Dados</string>
   </property>
   <widget class="QLabel" name="LabelCodeFactory">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Código:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelNameFactory">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Razaão Social:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelShortNameFactory">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome Fantasia:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="TButtonSaveFactory">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>50</y>
      <width>90</width>
      <height>80</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Salvar os dados para o Formulário...</string>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/images/PNG/48x48/Filesave-48x48.png</normaloff>:/images/PNG/48x48/Filesave-48x48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="CBoxCodeFactory">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Informar um Código para o Fabricante...</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEditIDFactory">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEditNameFactory">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>400</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Informar a Razão Social/Nome para o Fabricante...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEditShortNameFactory">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Informar o Nome Fantasia/Sigla/Apelido para o Fabricante...</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="GBoxFactoryStatusActive">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>171</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string>Status</string>
    </property>
    <widget class="QCheckBox" name="CheckBoxFactoryStatusActive">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Permite Ativar o Fabricante para ser Usado...</string>
     </property>
     <property name="text">
      <string>Ativo</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GBoxView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>731</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Fabricantes</string>
   </property>
   <widget class="QTableView" name="TableViewItemFactory">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>591</width>
      <height>261</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Fabricantes Atuais...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="TButtonRemoveFactory">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>160</y>
      <width>90</width>
      <height>80</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Fazer Remoção do Item selecionado da Tabela...</string>
    </property>
    <property name="text">
     <string>Remover</string>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/images/PNG/48x48/delete-48x48.png</normaloff>:/images/PNG/48x48/delete-48x48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="TButtonCleanFactory">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>90</width>
      <height>80</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Permite Limpar os Campos de Cadastro e Atualizar a Janela de Manutenção de Fabricantes...</string>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/images/PNG/48x48/Rebuild-48x48.png</normaloff>:/images/PNG/48x48/Rebuild-48x48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="TButtonExitFactory">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>90</width>
     <height>80</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Permite Sair da Janela de Manutenção de Fabricantes...</string>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/PNG/48x48/Exithome-48x48.png</normaloff>:/images/PNG/48x48/Exithome-48x48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
